src/index.js

```diff
@@ -24,6 +24,9 @@ export class BlockSuggestor {
    * @param {number} numBlocksPerCategory the size of each toolbox category
    */
   constructor(numBlocksPerCategory) {
+    /**
```

Comment:
This needs to be pushed back a space as per the lint comment below at https://github.com/jschanker/suggested-blocks-blockly/pull/8/commits/c763b7345d9e8005042887c6c858e906150e1bf6 (`Expected JSDoc block to be aligned`). You can see the other pending lint issues to fix.